application/src/main/java/org/togetherjava/tjbot/commands/filesharing/GistResponse.java

```diff
@@ -0,0 +1,23 @@
+package org.togetherjava.tjbot.commands.filesharing;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * @see <a href="https://docs.github.com/en/rest/gists/gists#create-a-gist">Create a Gist via
+ *      API</a>
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+final class GistResponse {
+    @JsonProperty("html_url")
+    private String htmlUrl;
+
+    public @NotNull String getHtmlUrl() {
+        return this.htmlUrl;
+    }
+
+    public void setHtmlUrl(@NotNull String htmlUrl) {
+        this.htmlUrl = htmlUrl;
+    }
```

Comment:
this method is never used in your code, so u can remove it

Reply:
I thought jackson uses it for creating the json response or smth

Reply:
hmm yea makes sense, when the field is not public